sl/content.md

```diff
@@ -0,0 +1,89 @@
+
+## Container Tags
+
+A tag will be provided for each of the [maintained major releases](http://www.scientificlinux.org/downloads/sl-versions/) of Scientific Linux (`6`, `7`).
+
+The `latest` tag will track the highest version numbered release of Scientific Linux.
+
+Updated images will be released roughly once per month.
+
+## Build Logs
+
+The SL docker containers are built on our internal build system and packaged at [Docker Cloud](https://cloud.docker.com/app/scientificlinux/repository/docker/scientificlinux/sl/builds/).
+
+## Getting Help
+
+* [SL Faq](https://www.scientificlinux.org/documentation/faq)
+
+* [Email Lists](https://www.scientificlinux.org/community)
+
+## Docker, overlayfs, and yum
+
+Recent Docker versions support the [overlayfs](https://docs.docker.com/engine/userguide/storagedriver/overlayfs-driver/) backend, which is enabled by default on most distros supporting it from Docker 1.13 onwards. On SL 6 and 7, that backend requires `yum-plugin-ovl` to be installed and enabled, which it is in our containers. Make it sure you retain the `plugins=1` option in `/etc/yum.conf` if you update that file; otherwise, you may encounter errors related to rpmdb checksum failure - see [Docker ticket 10180](https://github.com/docker/docker/issues/10180) for more details.
+
+## Package docs and licence files
+
+By default the SL docker images do not include these files.  If you require them, please remove `tsflags=nodocs` from `/etc/yum.conf` and run `yum reinstall mypackage` to recieve the documentation.
+
+---
+# Quick Reference
+
+## Example Usage
+You can try out the containers via:
+
+```
+$ docker pull scientificlinux/sl
+$ docker run -it %%IMAGE%%:6 cat /etc/redhat-release
+$ docker run -it %%IMAGE%%:7 cat /etc/redhat-release
+```
+
+## Enabling systemd in SL7
+The SL7 docker container ships with systemd mostly functional.  You can build a SL7 systemd enabled container with the following Dockerfile
+
+In order to run a container with systemd, you will need to mount the cgroups volumes from the host.
+
+```
+# Example SL7 systemd Dockerfile
+FROM %%IMAGE%%:7
+ENV container docker
+### To enable apache within the container uncomment the next two lines
+# RUN yum -y install httpd; yum clean all; systemctl enable httpd.service
+# EXPOSE 80
+### To enable apache within the container uncomment the previous two lines
+VOLUME [ "/sys/fs/cgroup" ]
+CMD ["/usr/sbin/init"]
+```
+
+You can build and run this example (with apache) via:
+```
+$ docker build --rm -t local/mycontainer
+$ docker run -ti -v /sys/fs/cgroup:/sys/fs/cgroup:ro -p 80:80 local/mycontainer
+```
+Which will run systemd within the container in a limited context.
+
+It is recommended that you install any relevant [OCI hooks](https://www.opencontainers.org/) for your container host - such as `oci-register-machine` or `oci-systemd-hook`.
+
+Some container hosts must add `-v /tmp/$(mktemp -d):/run` to the `docker run` command.
+
+---
+# About Scientific Linux
```

Comment:
I would put this "About SL" section first, but it is up to you. 

In any case, the resulting docs will have two generated sections added to the beginning:

- Supported tags and respective Dockerfile links (ex [arango](https://github.com/docker-library/docs/tree/3383325346b0ac532752d2f3725d1c654c3a3227/arangodb#supported-tags-and-respective-dockerfile-links))
- Quick Reference (ex [arango](https://github.com/docker-library/docs/tree/3383325346b0ac532752d2f3725d1c654c3a3227/arangodb#quick-reference))

And then the "License" section added to the end (ex. [arango](https://github.com/docker-library/docs/tree/3383325346b0ac532752d2f3725d1c654c3a3227/arangodb#license)).